src/components/paint-editor/paint-editor.css

```diff
@@ -126,7 +126,7 @@ $border-radius: 0.25rem;
 
 .mode-selector {
     display: flex;
-    max-width: 6rem;
+    max-width: 7.8rem;
```

Comment:
I wasn't able to see a situation where this was necessary, but it doesn't seem to hurt anything, either

Reply:
We looked at this together and came up with a better solution!